packages/client/modules/meeting/components/ReflectTemplateListOrg.tsx

```diff
@@ -77,6 +97,21 @@ const ReflectTemplateListOrg = (props: Props) => {
   useActiveTopTemplate(edges, activeTemplateId, teamId, true, 'retrospective')
 
   if (edges.length === 0) {
+    if (tier === 'personal' && featureFlags.templateLimit) {
```

Comment:
A use case to consider:
- A free user has already created a custom Team template
- We implement the template limit restriction
- They can still see their custom template in the Team tab
- The free user goes to the Organization tab and sees the message `Upgrade to Pro to create custom templates for your organization` even though they've already created a custom template

The downside of this approach is the user could feel confused by the message as they previously created a custom template. However, they would encounter this message anyway if they click "Create New Template". The upside is we share the upgrade to pro info without the user needing to click "Create New Template".

Alternatively, we could:

1. In this file, check whether the team has created any Team templates. If they have, show the "No other teams in your org are sharing a template" message
2. Prevent free users from having access to previously created custom templates. That would be a bad experience for teams that have invested time into creating them

I think the current implementation or option 1 works

Reply:
I personally feel that current option is ok, he already created, but now there came a certain limit.